content/docs/08-migration-guides/00-versioning.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Versioning
+description: Understand how the AI SDK approaches versioning.
+---
+
+# Versioning
+
+Each version number follows the format: `MAJOR.MINOR.PATCH`
+
+- **Major**: Breaking API updates that require code changes.
+- **Minor**: Blog post that aggregates new features and improvements over a period of time into a public release that highlights benefits to users.
+- **Patch**: New features and bug fixes.
+
+## API Stability
+
+We communicate the stability of our APIs as follows:
+
+### Stable APIs
+
+By default, all APIs without special prefixes are considered stable and ready for production use. We maintain backward compatibility for these features and only introduce breaking changes in major releases.
```

Comment:
Rm `By default`

Reply:
`We maintain backward compatibility for stable features and only introduce breaking changes in major releases.`